pkg/templates/canal/prerequisites.go

```diff
@@ -210,6 +216,7 @@ func calicoClusterRole() *rbacv1.ClusterRole {
 	}
 }
 
+// TODO(xmudrii): This doesn't exist anymore in the original manifest.
```

Comment:
@kron4eg That ClusterRole is not present in the manifest anymore and I don't know who creates that ServiceAccount. Should we drop this role?

Reply:
Yeah, probably yes.

Yet we don't have effective mechanism to delete "old stuff" currently.  Something like `kubectl apply --prune -l app=kubeone` would do. Probably we need to work on implementing it.